router/router.go

```diff
@@ -123,7 +116,14 @@ func (router Router) Can(u models.User, permissions []string) bool {
 	return isAllowed
 }
 
-// Execute calls the Plugin function provided
-func (route MentionRoute) Execute(api slack.Client, router Router, ev slackevents.AppMentionEvent, message string) {
-	route.Plugin(router, route.Route, api, ev, message)
+// AddMentionRoute sets upserts and element into `MentionRoutes` whose key is the provided `Name` field
+func (router Router) AddMentionRoute(route MentionRoute) {
+	router.MentionRoutes[route.Name] = route
+}
+
+// AddMentionRoutes calls `AddMentionRoute()` for each element in `routes`
```

Comment:
Your comment doesn't match what you actually do here. It is a good idea to just defer to `AddMentionRoute(route)` rather than duplicating its logic.